packages/skill-api/src/core/services/process-sanity-webhooks.ts

```diff
@@ -31,23 +30,23 @@ export async function processSanityProductsWebhook({
           body: 'Invalid webhook signature',
         }
       }
-      const product = req.body
-      console.log({product, req, params})
-      if (process.env.INNGEST_EVENT_KEY && product) {
+      const data = req.body
+
+      if (process.env.INNGEST_EVENT_KEY && data) {
         const inngest = new Inngest({
           id:
             process.env.INNGEST_APP_NAME ||
             process.env.NEXT_PUBLIC_SITE_TITLE ||
             'Sanity Products Webhook',
           eventKey: process.env.INNGEST_EVENT_KEY,
         })
-        console.log('sending inngest event')
-        await inngest.send({
-          name: SYNC_SANITY_PRODUCT,
-          data: {
-            product: req.body,
-          },
-        })
+        console.log('sending inngest event', data)
+        if (data._type === 'product') {
+          await inngest.send({
+            name: SYNC_SANITY_PRODUCT,
```

Comment:
the way to do this is to forward an "event type" and not build a stack of specific events so this handler ends up being a simple relay similar to stripe or mux

this means on the sanity side there will be a `type` at the top of the event like "type": "sanity/" + _type + "." + operation 


so then you can just send that out instead of conditionals


as an aside with events like this is always "what has occurred" not "what you want to happen" with event names. So "product created" or "product updated" would be the event name

Reply:
this is a good example of forwarding the event along and letting inngest do the filtering/conditionals https://github.com/joelhooks/course-builder/blob/main/apps/course-builder-web/src/app/api/mux/webhook/route.ts

Reply:
the principle here is that we want to edit `skill-api` package "practically never"